crates/topos-node/build.rs

```diff
@@ -0,0 +1,23 @@
+use std::process::Command;
+
+const DEFAULT_VERSION: &str = "detached";
+
+fn main() {
+    // Set TOPOS_VERSION to HEAD short commit hash if None
```

Comment:
```suggestion
    // Set TOPOS_VERSION to HEAD short commit hash unless it's already set
```

Reply:
Is this env variable typically set to something other than the short hash? What do you think about not making this configurable and just impose the short hash?

Reply:
This could be the `v0.0.XX` version but we didn't update the build process to make it passes through I think

Reply:
I think having the short hash is so useful so often that it's worth having it always. When we pass through the `x.y.z` string I still think we should include the short hash in the version string, e.g. `x.y.z-abc123abc`